### ISSUE TITLE
Doesn't work with cutscene (GBFR 1.2.1)

### ISSUE THREAD

**Issue by bnthomason** [ghostinthecamera/IGCS-GITC#30]
I just tried this camera, but for some reason in Grand Blue Fantasy Relink version 1.2.1 (says version 1.3.2 in game but that's a misnomer, as I downloaded an older version with depotdownloader, version 1.2.1 I believe), I cannot change the camera angle or positioning during unrendered battle cutscenes (such as boss intro and exit). Why is this and is there a workaround?

**Comment by bnthomason** [comment]
Something interesting I found

Camera still works with game version 1.3.2, but cutscene movement during certain cutscenes (boss intro, exit) don't. Not sure if it was working in 1.2.1, but can't verify as both the 1.3.2 game and vanilla, as well as 1.2.1 depot downloads (steam console and depotdownloader) all show 1.3.2 game version, with patch info and everything.

I think what is happening is that the creators of the game patched every single version of the depots and manifests with the latest version of the manifest.

**Comment by ghostinthecamera** [comment]
Hi, is it just the camera movement? everything else works - i.e. timestop etc

**Comment by ghostinthecamera** [comment]
I just checked this with Furycane, and everything was working. Can you tell me which boss intro/exit it is? Or better yet, can you send me a save file so i can check it as Furycane is the only boss i have access to on my saves

**Comment by bnthomason** [comment]
Yes, I'll send you my save. Please check with the various bosses I've unlocked (i.e. the end-game dungeon bosses (meant for multiplayer and sometimes solo with with in game characters), not just the story cutscenes bosses)

Specifically, I've tried it with Thunderbolts and Lightning first before I noticed the problem was all around with undertaking boss quests via the quest counter.

Here's my save

[GBFR saves.zip](https://github.com/user-attachments/files/16135049/GBFR.saves.zip)

**Comment by ghostinthecamera** [comment]
Ah no dice, can't get the save file to be recognised. It shows as corrupted. Unfortunately, I will only be able to check this if/when i play the game more

**Comment by bnthomason** [comment]
Well, you have to hex edit the save, now that I think about it, to match your steam id.

[https://youtu.be/d9Z5dJUgm_w](url)

Here's a quick and descriptive tutorial on how to do it without hex editing.

Let me know if this helped!

**Comment by bnthomason** [comment]
Hey, have you had time to edit the save data? I presume you have a busy schedule, so I'll wait patiently. Keep me updated!

**Comment by ghostinthecamera** [comment]
Should be fixed in the new tools being released soon
